### PR TITLE
Added test case that gets host OS version of the image

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

**Comment by horia-delicoti** [comment]
@lifeeth @villaseca

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ output.xml
 report.html
 env.list
 resin.img
+/tmp
```
Comment (lifeeth) [inline .gitignore]: This doesnt seem to be needed unless we in / - In which case we have a lot more things to cleanup.

File: qemux86-64.robot [hunk 2]
```diff
@@ -33,6 +33,14 @@ Check host OS fingerprint file
   Verify resin-root.fingerprint in "${path_to_fingerprint}"
   Unmount "${mount_destination}"
   Detach loop device "${path_to_loop}"
+Get host OS version of the image
+  Set up loop device for "${image}"
+  Find the last loop device that was assigned
+  Set Suite Variable    ${path_to_loop}    /dev2/loop${LOOPDEVICE}
+  Mount "${path_to_loop}p${host_os_partition}" on "${mount_destination}"
+  Get the host OS version of the image
```
Comment (lifeeth) [inline qemux86-64.robot]: Instead of setting the os_version Suite variable inside the keyword - lets set it here?

File: qemux86-64.robot [hunk 3]
```diff
@@ -33,6 +33,14 @@ Check host OS fingerprint file
   Verify resin-root.fingerprint in "${path_to_fingerprint}"
   Unmount "${mount_destination}"
   Detach loop device "${path_to_loop}"
+Get host OS version of the image
+  Set up loop device for "${image}"
```
Comment (lifeeth) [inline qemux86-64.robot]: Coming to think of it - should we merge 

Set up loop device for "${image}" and Find the last loop device that was assigned to one keyword which has a behavior as follows

${LOOPDEVICE} = Set up loop device for "${image}"

**Pull request merged**